react/src/lib/components/WellLogViewer/MapAndWellLogViewer.tsx

```diff
@@ -138,11 +140,26 @@ export class MapAndWellLogViewer extends React.Component<Props, State> {
 
         this.onMouseEvent = this.onMouseEvent.bind(this);
     }
-    componentDidUpdate(prevProps: Props /*, prevState: State*/): void {
+    componentDidUpdate(prevProps: Props, prevState: State): void {
         if (this.props.editedData !== prevProps.editedData) {
             this.setState({ editedData: this.props.editedData });
             0;
         }
+        console.log(
```

Comment:
Please remove or use `console.debug` if this is useful.

Reply:
console.log removed